src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export function getVisibleMovies(
+  movies: Movie[],
+  query: string,
+) {
+  const normalQuery = query.toLowerCase();
+
+  // eslint-disable-next-line max-len
+  return movies.filter(movie => movie.title.toLowerCase().includes(normalQuery) || movie.description.toLowerCase().includes(normalQuery));
```

Comment:
`movie.title.toLowerCase().includes(normalQuery)` - move to variable
`movie.description.toLowerCase().includes(normalQuery)` - move to variable